lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const _ = require('lodash');
+const traverse = require('traverse');
+
+module.exports = {
+	meta: {
+		docs: {
+			description: 'Should contain a custom error message for multiple assertions.',
+			category: 'Possible Errors',
+			recommended: true
+		},
+		schema: []
+	},
+	create: context => {
+		return {
+			'ExpressionStatement': node => {
+
```

Comment:
Normally I'd like to extract this out into a helper method, but I think it'd hurt code readability in this case...